src/handlers.ts

```diff
@@ -116,10 +116,7 @@ export const postFeedback = async (ctx: IRouterContext): Promise<any> => {
   }
 
   try {
-    const { status_code, status_message, url } = await twitter.tweetStatus(params)
-    if (status_code !== 200) {
-      throw new Error(`${status_code} ${status_message}`)
-    }
+    const { url } = await twitter.tweetStatus(params)
     return Object.assign(ctx.response, { status: 201, body: { url } })
   } catch (error) {
```

Comment:
Let's also get rid of the catch error here, which would prevent the middleware from catching past errors